README.md

```diff
@@ -19,17 +30,27 @@ Providing information to the customer increases transparency and reduces communi
 
 The interactions should not refresh the page.
 
-## Technology requirements
-
-You can use any libraries, task runners and build processors. **React** and **JavaScript** are the only hard requirements. ES6 and TypeScript are highly encouraged, as well as MST and MobX if needed.
+# Technology requirements
 
-**Submissions using view libraries/frameworks other than React will not be accepted.**
+**React** and **JavaScript** are mandatory requirements. Apart from this,  you can use any libraries, task runners and build processors. ES6 and TypeScript are highly encouraged, as well as MST and MobX if needed.
```

Comment:
```suggestion
**React** and **JavaScript** are mandatory requirements. Apart from this, you can use any libraries, task runners and build processors. ES6 and TypeScript are highly encouraged.
```

I think we should include the part about not accepting non-React solutions. Also I don't think we should suggest MobX and MST anymore.